patches/minecraft/net/minecraft/client/renderer/model/BlockModel.java.patch

```diff
@@ -78,3 +78,20 @@
     public boolean func_178300_b(String p_178300_1_) {
        return !MissingTextureSprite.func_195675_b().equals(this.func_228816_c_(p_178300_1_).func_229313_b_());
     }
+@@ -267,6 +290,16 @@
+       return this.field_178317_b;
+    }
+ 
++   // Forge start
++   public BlockModel.GuiLight getGuiLightWithFallback(BlockModel.GuiLight fallback) {
```

Comment:
Perhaps this should go in an extension interface?

Reply:
Tought about it, but that would require ATing the field public. If you think that's better than the patch I'll do that.

Reply:
Done.